### ISSUE TITLE
max length seems too long

### ISSUE THREAD

**Issue by Xoey0314** [chhylp123/hifiasm#719]
Hi,

When I only use HiFi data to assemble the genome by hifiasm before ragtag, the max length of contigs is 91Mb. But the max length of the chr is usually 60Mb. I want to know it has erro or no. 

Best wishes,
Xoey

**Comment by chhylp123** [comment]
It might be a misassembly that joins two chromosomes.

**Comment by Xoey0314** [comment]
yes,I want to know how to solve this problem by hifiasm-self. I try add l0, the max length was reduced to 71Mb, but it's still long.
